apis/frontend/implementation-guide-session-api/parameterless-attribution.md

```diff
@@ -1,122 +0,0 @@
----
-description: >-
-  This documentation highlights the differences in the response rendered when
-  using Parameterless Attribution with Session API
----
-
-# Parameterless Attribution
```

Comment:
Since parameterless attribution is becoming the norm HTML campaign injection should always be delegated to Nosto, so this page becomes obsolete